src/visualization/visualize_treatment_models.py

```diff
@@ -1,5 +1,5 @@
 """
-Visual explanations of the models
+Visual explanations of the models.
 """
 import os
 import matplotlib.pyplot as plt
```

Comment:
totally cool with me that you are using matplotlib here, just curious why you didn't use plotly?

Reply:
I initially had issues with plotly and saving static plots as a file, and had run into this issue in the google colab notebook too. This is one of the topics for the broader testing conversation that would be good to have soon.